### ISSUE TITLE
Probes visualization features

### ISSUE THREAD

**Issue by hernanmd** [cormas/cormas#343]
From #341 

Probes visualization window should have more features (choosing the probes to be displayed.

@SergeStinckwich may you detail which features?

**Comment by pierbo** [comment]
Yes, it is possible. Yesterday with Hernan we had a short meeting. In fact, there was an inversion in the possibility of selecting several elements of a list: only one method of initialization and one method of step should be chosen, but it should be possible to select several probes. This will be quickly corrected

**Comment by nbecu** [comment]
Yes. Of course

It is possible to have several proves on the same figure, to configure the
x and y scales, to change the color of eah probe and as well to compare the
results with the one of a previously recorded simulation

Le ven. 12 mars 2021 à 11:57, Etienne DELAY ***@***.***> a
écrit :
